server/routes/report.ts

```diff
@@ -230,16 +222,41 @@ export default function (router: IRouter) {
       }
     }
   );
-}
 
-function parseEsErrorResponse(error: any) {
-  if (error.response) {
-    try {
-      const esErrorResponse = JSON.parse(error.response);
-      return esErrorResponse.reason || error.response;
-    } catch (parsingError) {
-      return error.response;
+  // Delete single report by id
+  router.delete(
+    {
+      path: `${API_PREFIX}/reports/{reportId}`,
```

Comment:
Think this route should be something like `${API_PREFIX}/delete_report_by_id/{reportId}`

Reply:
https://restfulapi.net/resource-naming/
> URIs should not be used to indicate that a CRUD function is performed. URIs should be used to uniquely identify resources and not any action upon them. HTTP request methods should be used to indicate which CRUD function is performed.

Reply:
TIL